ambry-cloud/src/main/java/com/github/ambry/cloud/azure/ADAuthBasedStorageClient.java

```diff
@@ -46,9 +46,9 @@
  */
 public class ADAuthBasedStorageClient extends StorageClient {
   private static final String AD_AUTH_TOKEN_REFRESHER_PREFIX = "AdAuthTokenRefresher";
-  private final ScheduledExecutorService tokenRefreshScheduler =
+  private ScheduledExecutorService tokenRefreshScheduler =
       Utils.newScheduler(1, AD_AUTH_TOKEN_REFRESHER_PREFIX, false);
```

Comment:
Are you sure you want to schedule the refresher here as well as in buildBlobServiceClient?  Won't you end up with two of them?

Reply:
fixed it.